src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        startDuke();
+
+        Scanner userInput = new Scanner(System.in);
+        String line;
+        ArrayList<Task> tasks = new ArrayList<Task>();
+
+        line = userInput.nextLine();
+        while (!"bye".equals(line)) {
+            Parse parsedInput = new Parse(line);
```

Comment:
i like that you use a class for parsing inputs and creating your own methods for convenience.